test/tests/rails4.rb

```diff
@@ -1068,6 +1068,18 @@ def test_ssl_verification_bypass
       :user_input => nil
   end
 
+  def test_ssl_verification_bypass_in_new
+    assert_warning :type => :warning,
+      :warning_code => 71,
+      :fingerprint => "6e5454d43e4eb095da16d5b9e65a48d9ea55fac3e1fd2c6ae740e74fe1ad581c",
+      :warning_type => "SSL Verification Bypass",
+      :line => 24,
+      :message => /^SSL\ certificate\ verification\ was\ bypasse/,
+      :confidence => 0,
+      :relative_path => "app/controllers/application_controller.rb",
```

Comment:
Are the relative_path and line numbers off here?  (the bad code sample is in `sweet_lib`, not `application_controller`)